src/components/Table/GridTable.tsx

```diff
@@ -973,10 +973,12 @@ export type GridCellContent = {
   indent?: 1 | 2;
   colspan?: number;
   typeScale?: Typography;
-  /** Allows the cell to stay in place when the user scrolls horizontally */
+  /** Allows the cell to stay in place when the user scrolls horizontally, i.e. frozen columns. */
   sticky?: "left" | "right";
   /** If provided, content of the cell will be wrapped within a <button /> or <a /> tag depending on if the value is a function or a string. */
   onClick?: () => {} | string;
+  /** Custom css to apply directly to this cell, i.e. cell-specific borders. */
+  css?: Properties;
```

Comment:
I feel like this pained you to put in. 😂

Reply:
@bdow well, it's this or `nth-of-type` selectors to do cell borders, so at the end of the day I dislike those _worse_ :-D